controllers/profiles/getOtherProfileByUsername.js

```diff
@@ -30,49 +30,45 @@ module.exports = async (req, res) => {
         status: 'error',
         message: 'User not found',
       });
-    } else {
-      let copyUser = { ...user.dataValues };
-      delete copyUser.following;
-      delete copyUser.follower;
+    }
+    let copyUser = { ...user.dataValues };
+    delete copyUser.following;
+    delete copyUser.follower;
 
-      copyUser.following = user.dataValues.following.length;
-      copyUser.following_symbol = checkMoreOrLess(
-        user.dataValues.following.length
-      );
-      copyUser.follower = user.dataValues.follower.length;
-      copyUser.follower_symbol = checkMoreOrLess(
-        user.dataValues.follower.length
-      );
+    copyUser.following = user.dataValues.following.length;
+    copyUser.following_symbol = checkMoreOrLess(
+      user.dataValues.following.length
+    );
+    copyUser.follower = user.dataValues.follower.length;
+    copyUser.follower_symbol = checkMoreOrLess(user.dataValues.follower.length);
 
-      let findIndex = user.dataValues.follower.findIndex(
+    let findIndex = user.dataValues.follower.findIndex(
+      (value) => value.user_id_follower === req.userId
+    );
+    const isUserFollowingMe = user.dataValues.following.findIndex(
+      (value) => value.user_id_followed === req.userId
+    );
+
+    copyUser.is_following = findIndex > -1 ? true : false;
+    copyUser.isSignedMessageEnabled = isUserFollowingMe > -1 ? true : false;
```

Comment:
mas @falhar isSignedMessageEnabled also should check with onlyReceivedAnonDmFromUserFollowing
so should be like this
` copyUser.isSignedMessageEnabled = isUserFollowingMe > -1 && onlyReceiveedAnonDmFromUserFollowing;`

Reply:
Mas @falhar also please isSignedMessageEnabled on this endpoint
`/get-profile/:username`